.vscode/launch.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Can you please remove your vscode folder from the PR?

Reply:
Yep, done. Getting the launch configs right was non-trivial, so once the dust settles I can PR a change to the docs with an example for other folks who want to run tests in VSCode.